<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calculo</class>
 <widget class="QWidget" name="calculo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Courier&quot;;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Equação</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>x0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>y0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Passo(h)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Repetições</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Courier&quot;;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEquacao"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineX"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineY"/>
      </item>
      <item>
       <widget class="QLineEdit" name="linePasso"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineRepeticao"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="buttonCalcular">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>10</horstretch>
          <verstretch>10</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Courier&quot;;
background: rgb(132, 132, 132)</string>
        </property>
        <property name="text">
         <string>Calcular</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonLimpar">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Courier&quot;;
background: rgb(132, 132, 132)</string>
        </property>
        <property name="text">
         <string>Limpar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
